src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1741,13 +1742,41 @@ source = do
 linkTitle :: MarkdownParser String
 linkTitle = quotedTitle '"' <|> quotedTitle '\''
 
-link :: MarkdownParser (F Inlines)
-link = try $ do
+wikilinkInfo :: MarkdownParser (F Inlines, String)
+wikilinkInfo = do
+  string "[["
+  (_, raw) <- withRaw $ charsInBalancedBrackets 2
+  guard $ not $ null raw
+  let deunder c = if c == '_' then ' ' else c
+      deiki s = map deunder $ (reverse . takeWhile (/= '/') . reverse) $ s
```

Comment:
If you are like me and like [pointfree style](https://wiki.haskell.org/Pointfree)point-less) this could be written as

    deiki = map deunder . reverse . takeWhile (/= '/') . reverse